packages/hemera-mongo-store/store.js

```diff
@@ -17,9 +28,16 @@ class MongoStore extends Store {
    *
    * @memberOf MongoStore
    */
-  constructor (driver, options = {}) {
-    options.mongo = {}
+  constructor (driver,options = { mongo: {}, store: {} }) {
     super(driver, options)
+    if (!this.options.hasOwnProperty('store')) {
```

Comment:
Yes but its up to the developer to pass valid options. I think passing undefined as mongodb option is valid.